docs/releases/changelog-dev.md

```diff
@@ -2,6 +2,20 @@
 
 ### New features since last release
 
+- Allow execution of `QProgram` through `platform.execute_qprogram` method for Quantum Machines hardware.
+  [#648](https://github.com/qilimanjaro-tech/qililab/pull/648)
+
+### Improvements
+
+- Added `bus_mapping` parameter in `QbloxCompiler.compile` method to allow changing the bus names of the compiled output.
```

Comment:
I don't know if the standard is to do what you did here - putting different Improvements / Features belonging to the same PR separately - or putting them together (just wondering if there's a standard at all)